Jenkinsfile

```diff
@@ -40,5 +40,5 @@
 // I.e. for testing library changes
 //@Library(value="pipeline-lib@your_branch") _
 
-packageBuildingPipeline(['distros' : ['centos7', 'centos8', 'leap15']])
+packageBuildingPipelineDAOSTest(['distros' : ['el8', 'leap15']])
```

Comment:
The idea for this is to use a test tag for mpifileutils tests to be tested. Something like:
https://github.com/daos-stack/hdf5-vol-daos/blob/master/Jenkinsfile#L43
but different tag of course

Reply:
Ah, yes indeed!  Great catch @mchaarawi!

@daltonbohning You'd want to use a tag that (at least) captures the (i.e. weekly) test that is failing.  You can use more than one tag IIRC, just like `Test-tag:`